web/modules/custom/collection/config/install/core.entity_view_display.rdf_entity.collection.view_mode_tile.yml

```diff
@@ -32,8 +32,10 @@ targetEntityType: rdf_entity
 bundle: collection
 mode: view_mode_tile
 content:
-  field_ar_description:
-    weight: 2
+  field_ar_abstract:
```

Comment:
For now it I think that we should not remove abstract from a tile view mode, because it is not clear if there will be featured view mode or maybe featured tiles will be done in a different way. For now abstract is not displayed in hidden view mode:
`{% if view_mode == 'view_mode_featured' %}`